sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -318,5 +368,17 @@ public virtual async Task<Response<AccessToken>> GetTokenForTeamsUserAsync(GetTo
                 throw;
             }
         }
+
+        private static int? GetTokenExpirationInMinutes(TimeSpan tokenExpiresIn, string paramName)
+        {
+            try
+            {
+                return tokenExpiresIn == default ? null : Convert.ToInt32(tokenExpiresIn.TotalMinutes);
+            }
+            catch (OverflowException ex)
+            {
+                throw new ArgumentException($"The {paramName} argument is out of permitted bounds [1,24] hours. Please refer to the documentation and set the value accordingly.", paramName, ex);
```

Comment:
ArgumentOutOfRangeException is more suitable
https://learn.microsoft.com/en-us/dotnet/api/system.argumentoutofrangeexception?view=net-6.0

Reply:
Updated.